<ui version="4.0" >
 <class>dialog</class>
 <widget class="QDialog" name="dialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>488</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>99999</width>
    <height>999999</height>
   </size>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>Pick shotgun playlist</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Department</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_dept" >
         <item>
          <property name="text" >
           <string>anim</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>lens</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>light</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>alldepts</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>999999</width>
         <height>999999</height>
        </size>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>200</red>
             <green>200</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>93</red>
             <green>93</green>
             <blue>93</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>139</red>
             <green>139</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="AlternateBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>200</red>
             <green>200</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>200</red>
             <green>200</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>93</red>
             <green>93</green>
             <blue>93</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>139</red>
             <green>139</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="AlternateBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>200</red>
             <green>200</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>200</red>
             <green>200</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>93</red>
             <green>93</green>
             <blue>93</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>139</red>
             <green>139</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>139</red>
             <green>139</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="AlternateBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>200</red>
             <green>200</green>
             <blue>200</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip" >
        <string>Doubleclick the row to load the playlist</string>
       </property>
       <property name="frameShape" >
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="showDropIndicator" stdset="0" >
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="gridStyle" >
        <enum>Qt::DotLine</enum>
       </property>
       <property name="sortingEnabled" >
        <bool>true</bool>
       </property>
       <column>
        <property name="text" >
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>DPT</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>NAME</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>DATE_TIME</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>DESCRIPTION</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" >
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_button" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ok_button" >
         <property name="text" >
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
